project.godot

```diff
@@ -14,6 +14,10 @@ config/name="Godot Community Project 1"
 config/features=PackedStringArray("4.3", "Forward Plus")
 config/icon="res://icon.svg"
 
+[dotnet]
```

Comment:
Are you using godot .net? I am unsure if this will cause issues.

Reply:
yes I am, as I usually develop in c# but I will switch to the regular hodot version now.